### DISCUSSION TITLE
My sql data is not getting retrieved and there is a problem with my DNS

### DISCUSSION THREAD

**Discussion by SravyaSuravarjula** [learndevopsonline/learndevopsonline#1576] [Q&A] [answered]
![Screenshot (42)](https://github.com/learndevopsonline/learndevopsonline/assets/149948281/26521f38-65a4-4e72-9c0a-fbd743d25ac9)
![Screenshot (43)](https://github.com/learndevopsonline/learndevopsonline/assets/149948281/df8acedb-2b68-4594-8aec-b158aa7ee6d4)

**Comment by sreedharm07** [comment, marked as answer]
solved in lab session
